lib/views/pages/screens/profile_page.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/material.dart';
+
+Column profilePage() => Column(
```

Comment:
Keep this ui code in ProfileScreen widget. Only for profileItem have a separate file